Doc/library/sqlite3.rst

```diff
@@ -1442,6 +1442,12 @@ Cursor objects
       and there is no open transaction,
       a transaction is implicitly opened before executing *sql*.
 
+      .. versionchanged:: 3.12
```

Comment:
```suggestion
      .. deprecated-removed:: 3.12 3.14
```

Reply:
Are you sure about that? The API itself is not deprecated, we're just changing it. I'm not sure what's the recommended practice here.

Reply:
Yeah, I know what you mean. I feel like ideally this notice would be phrased along the lines of "X behaviour/practice is now deprecated" rather than "a DeprecationWarning is now emitted". That would be more to-the-point, and it would also work more naturally with this directive (X behaviour/practice is deprecated in 3.12, and will be removed entirely in 3.14).

But I was struggling to come to with a concrete suggestion for how to reword these notices :/

Reply:
(The whatsnew and NEWS entries look great btw, it's just the notices in the API docs that feel _slightly_ clunky to me)

Reply:
Yes, but documented as deprecated and emitting a `DeprecationWarning` are similar, but not equal, things 🙂 With the former, we don't need to emit a warning in the code.

Reply:
To your original question, btw: I think it's pretty standard to use `.. deprecated` or `.. deprecated-removed`, even if it's just a particular usage of an API, rather than the API itself. See e.g. https://docs.python.org/3.12/library/asyncio-policy.html#asyncio.DefaultEventLoopPolicy, where the directive is used even though the class itself hasn't been deprecated at all; or #19867, which deprecated just a specific parameter; or lots of other examples in our docs 🙂

Reply:
Aight! I'll try to reword it.

Reply:
I've updated it to use `deprecated-removed`, and I put in an extra line regarding what happens in 3.14. I'm too tired to reword the text 🙂